tradeexecutor/ethereum/execution_uniswap_v3.py

```diff
@@ -0,0 +1,339 @@
+import logging
```

Comment:
There is no need to create a separate execution module for Uniswap v3, because this is mostly copy-paste from the existing execution module.